### ISSUE TITLE
Send email notifications to the group admins of requests to join it

### ISSUE THREAD

**Issue by pandark** [inventaire/inventaire#274]
When someone requests to join a group of which you're an admin, you only see a notification on the website.
Being notified by email would help validate these requests more quickly.

https://inventaire.io/settings/notifications
> email me when
> - […]
> - Someone requests to join one of my groups

**Comment by jum-s** [comment]
When the `requestToJoin` function in [group_actions](/home/vinny/inventaire/docker/inventaire/client/app/modules/network/lib/group_actions.coffee#L29) is triggered, it seems that [`membership_actions`](https://github.com/inventaire/inventaire/tree/master/server/controllers/groups/lib/membership_actions.coffee#L20) emits a `group:requested` radio signal. 

[`server/lib/emails/mailer.coffee`](https://github.com/inventaire/inventaire/tree/master/server/lib/emails/mailer.coffee#L32) could catch the radio signal
```coffee
  radio.on 'group:requested', sendEmail.groupJoinRequest
```
and build a yet-to-create email template for `groupJoinRequest`

**Comment by maxlath** [comment]
@jum-s yes, the event handler could use the provided `groupId` to find the admins and send them a notification
